README.md

```diff
@@ -64,27 +83,11 @@ graph TD;
   id6-->id8
 ```
 
-With input file the `nyaml` converter checks for the correct file type and call appropriate converter. For XML file, the XML converter parse the `XML` file, by [lxml](https://lxml.de/) python library, into a `XML` tree object. By following the NXDL rules the converter writes the application definition or base class object into `yaml` file following the `nyaml` syntax. If the input file is `yaml` then the `yaml` converter collects the comments in a `Comments` object and parse the `yaml` file into python `dictionary` object. Later, the application definition or base classes will be written into `XML` file from the `Comments` and python `dictionary` object.
+The tool is a command line application which picks up provided input in either YAML or XML triggering a conversion vice versa automatically. Functionalities of the [lxml](https://lxml.de/) python library are used to process an `XML` tree object. By following the NXDL rules the converter writes the application definition or base class object into `yaml` file following the `nyaml` syntax. If the input file is `yaml` then the `yaml` converter collects the comments in a `Comments` object and parse the `yaml` file into python `dictionary` object. Later, the application definition or base classes will be written into `XML` file from the `Comments` and python `dictionary` object.
```

Comment:
Not a big fan of the first sentence, seems a bit verbose to me. I think combining the two drafts here would be good.